kafka-rest/src/test/java/io/confluent/kafkarest/unit/AdminClientWrapperTest.java

```diff
@@ -0,0 +1,35 @@
+package io.confluent.kafkarest.unit;
+
+import io.confluent.kafkarest.AdminClientWrapper;
+import io.confluent.kafkarest.KafkaRestConfig;
+import io.confluent.kafkarest.entities.Topic;
+import io.confluent.rest.RestConfigException;
+import org.apache.kafka.clients.admin.MockAdminClient;
+import org.apache.kafka.common.Node;
+import org.apache.kafka.common.TopicPartitionInfo;
+import org.junit.Test;
+
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Properties;
+
+import static org.junit.Assert.assertEquals;
+
+public class AdminClientWrapperTest {
+  @Test
+  public void testGetTopicOnNullLeaderDoesntThrowNPE() throws RestConfigException {
+    Properties props = new Properties();
+    props.put(KafkaRestConfig.KAFKACLIENT_INIT_TIMEOUT_CONFIG, "100");
+    KafkaRestConfig config = new KafkaRestConfig(props);
+    Node controller = new Node(1, "a", 1);
+
+    MockAdminClient adminClient = new MockAdminClient(Arrays.asList(controller, null), controller);
```

Comment:
Yeah, otherwise `addTopic` throws an exception - https://github.com/apache/kafka/blob/ed94cf4d314d1fea63fa4972e5aec73d35cd6813/clients/src/test/java/org/apache/kafka/clients/admin/MockAdminClient.java#L91

Reply:
ok.